### PR TITLE
[docs] Convert general WinC doc to Markdown

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

File: tools/ansible/docs/ocp-4-4-with-windows-server.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+Install git to clone github repository:
+
+```sh
+$ sudo dnf install git
+```
+
+## Bring up the OpenShift cluster with ovn-kubernetes
+
+Download the dev-preview 4.4.0-rc.3
+[OpenShift Installer and Client](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.4.0-rc.3/)
```
Comment (aravindhp) [inline tools/ansible/docs/ocp-4-4-with-windows-server.md:53]: Please point to [OpenShift 4.4.4](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.4/) here and in the other docs.

**Comment by codyhoag** [comment]
I will squash commits once reviews have finalized. I wanted to preserve the changes based off the original doc

**Comment by aravindhp** [comment]
@codyhoag, @Anandnatraj has approved so please squash your commits. I will approve and get it merged.

**Comment by codyhoag** [comment]
@aravindhp They're now squashed. Thanks!

**Pull request merged**